app/controllers/stock_assets_controller.rb

```diff
@@ -1,62 +0,0 @@
-# The stock assets controller makes stock assets for
```

Comment:
THis was used at the end of the old library creation pipelines to create stocks,